src/components/common/SimpleTable/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.table<{ $numColumns?: number }>`
+  font-size: var(--font-size-default);
+  background: var(--color-primary);
+
+  margin: 0;
+  height: calc(100% - 8.8rem);
+  padding: 0;
+  box-sizing: border-box;
+  width: 100%;
+  border-spacing: 0;
+  display: inline-grid;
+  grid-template-areas:
+    'head-fixed'
+    'body-scrollable';
+
+  tr {
```

Comment:
please change these to nested props:
```scss
tr {
  text-align: left;
  padding: 0;

  > td {
    padding: 0;
    ....
  }
```

makes it much easier to reason with and edit later